vpc-sample/main.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file holds functions for testing VPC ingress/egress with Cloud Functions
+
+# [START egress_hello_world]
```

Comment:
Does it matter that this sample is for both Functions and Cloud Run?  I'm building it with Cloud Run & Functions Framework here, but in the documentation I think it will be presented in both the context of Functions and Cloud Run.

Reply:
Fair point, for now let's prefix run bits with `run_` and functions bits with `functions_`.
If parts are deployable to both, double-wrap them like we did with the id token sample.